src/test/groovy/unit/org/pih/warehouse/order/OrderAdjustmentSpec.groovy

```diff
@@ -21,11 +22,11 @@ class OrderAdjustmentSpec extends Specification implements DomainUnitTest<OrderA
         orderAdjustment.invoiceable == isInvoiceable
 
         where:
-        isInvoiced | orderStatus         || isInvoiceable
-        true       | OrderStatus.PENDING || false
-        false      | OrderStatus.PLACED  || true
-        true       | OrderStatus.PLACED  || false
-        false      | OrderStatus.PENDING || false
+        orderStatus         | hasRegularInvoice  || isInvoiceable
+        OrderStatus.PENDING | false              || false
+        OrderStatus.PLACED  | false              || true
+        OrderStatus.PENDING | false              || false
```

Comment:
hasRegularInvoice should be true here right? Otherwise it;s the same as the first case